js/base/main.js

```diff
@@ -468,6 +468,18 @@ export class RipeCommonsMainPlugin extends RipeCommonsPlugin {
         // as this is going to be used by multiple parts
         Vue.component("global-events", GlobalEvents);
 
+        // configures a custom error handler for vue
+        Vue.config.errorHandler = (error, vm, info) => {

```

Comment:
just asking, can't this be also moved to  white's `_loadVue` in https://github.com/ripe-tech/ripe-white/blob/a32dbdc4d371bdcc16c40f67ae798145722b8e0e/js/plugins/base/main.js#L212?